pkgs/slurm-ops/src/slurm_ops/core/base.py

```diff
@@ -546,6 +551,11 @@ def path(self) -> Path:
         """Get the path to the `slurm.key` secret file."""
         return self._file
 
+    @path.setter
+    def path(self, new_path: str) -> None:
+        """Set the path to the `slurm.key` secret file."""
+        self._file = Path(new_path)
+
```

Comment:
Same as my comment above. BE GONE EXTRA LINES CODE; YOU'RE NOT A PRODUCTIVITY METRIC HERE.

```suggestion
    @path.setter
    def path(self, new_path: str) -> None:
        """Set the path to the `slurm.key` secret file."""
        self._file = Path(new_path)

```